snmp/datadog_checks/snmp/data/default_profiles/_cisco-ipsec-flow-monitor.yaml

```diff
@@ -0,0 +1,83 @@
+metrics:
+  - MIB: CISCO-IPSEC-FLOW-MONITOR-MIB
+    table:
+      name: cikeTunnelTable
+      OID: 1.3.6.1.4.1.9.9.171.1.2.3
+    symbols:
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.15
+        name: cikeTunLifeTime
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.19
+        name: cikeTunInOctets
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.20
+        name: cikeTunInPkts
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.21
+        name: cikeTunInDropPkts
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.27
+        name: cikeTunOutOctets
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.28
+        name: cikeTunOutPkts
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.29
+        name: cikeTunOutDropPkts
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.35
```

Comment:
This has a type TunnelStatus and I think it would be worth making this a tag and adding a mapping for it knowing that
```
TunnelStatus ::= TEXTUAL-CONVENTION
    STATUS          current
    DESCRIPTION
        "The status of a Tunnel.  Objects of this type may
        be used to bring the tunnel down by setting
        value of this object to destroy(2).  Objects of this
        type cannot be used to create a Tunnel."
    SYNTAX          INTEGER  {
                        active(1),
                        destroy(2)
                    }
```